homeassistant/loader.py

```diff
@@ -878,7 +926,9 @@ async def async_get_component(self) -> ComponentProtocol:
         self._component_future = self.hass.loop.create_future()
         try:
             try:
-                comp = await self.hass.async_add_import_executor_job(self.get_component)
+                comp = await self.hass.async_add_import_executor_job(
+                    ft.partial(self.get_component, preload_platforms=True)
```

Comment:
You can avoid the partial if you just pass True. Is it worth the extra wrapper ?

Reply:
Yeah that one was a bit autopilot. removed in 996b196369d

Reply:
good call 👍